rules/eslint/style/on.js

```diff
@@ -117,29 +117,29 @@ module.exports = {
     // disallow dangling underscores in identifiers
     "no-underscore-dangle": "off",
     // disallow the use of Boolean literals in conditional expressions
-    "no-unneeded-ternary": "off",
+    "no-unneeded-ternary": "error",
     // disallow whitespace before properties
-    "no-whitespace-before-property": "off",
+    "no-whitespace-before-property": "error",
     // enforce the location of single-line statements
-    "nonblock-statement-body-position": "off",
+    "nonblock-statement-body-position": ["error", "below"],
     // enforce consistent line breaks inside braces
-    "object-curly-newline": "off",
+    "object-curly-newline": ["error", { "minProperties": 2 }],
```

Comment:
Not sure I like this one. I would allow any number of props in an object that fit on a single line.